### ISSUE TITLE
NVDA on Windows reads accordion content multiple times and out of order

### ISSUE THREAD

**Issue by fpigeonjr** [uswds/uswds#4630]
### Describe the bug
NVDA on Windows reads accordion content multiple times and out of order.
VoiceOver on Safari reads as expected.

### Steps to reproduce the bug
1. [Visit preview site here](https://federalist-edd11e6f-8be2-4dc2-a85e-1782e0bcb08e.app.cloud.gov/site/gsa/usagov-benefits-eligibility/).
2. Open screen reader
3. Tab into accordion
4. Listen for content read inside the accordion
5. Hear error

### Expected behavior
Read content of accordion in the right order and not repeat.

### Screenshots

Accordion content reading is off. Not in correct order and content is repeated.
Using NVDA on Chrome on windows10.

https://user-images.githubusercontent.com/4629398/162072512-39493242-61a4-4720-bd1c-0640dcd337e4.mov

Accordion content reading as expected using VoiceOver on Safari

https://user-images.githubusercontent.com/4629398/162068483-5e4a81ca-6e49-4d93-9705-65907ca359c9.mov

### System setup
- **USWDS version:**2.13.1
- **Device:**Virtual machine 
- **Operating system:**Windows 10
- **Browser and version:**Chrome latest

### Additional context
[Repo here, if needed](https://github.com/GSA/usagov-benefits-eligibility)

**Comment by brunerae** [comment]
@mejiaj feedback from A11Y team:

> I ran through each accordion without that reported error. I tested on my PC in Chrome with NVDA. I did notice that the video of the error in GitHub looks different than the website provided. Has the website been updated or is it possibly a different link? The other thing I noticed was that the screen reader would state “graphic you did not provide information for” on each bullet within the accordion. I am not sure if this is intended behavior but figured I would flag it.

**Comment by brunerae** [comment]
This is ready to close as we were unable to reproduce the described bug.
